modules/op-release-notes-1-6.adoc

```diff
@@ -0,0 +1,262 @@
++
+To deploy on an infra node, you update the `TektonConfig` and `Subscription` CRs with the infra label as `nodeSelector` and `toleration` so that these labels are applied to the infra node.
+//  (link:https://issues.redhat.com/browse/SRVKP-1544[SRVKP-1544])
+
+[id="deprecated-features-1-6_{context}"]
+== Deprecated features
+
+//[id="deprecated-cli-0-21-0-release-1-6_{context}"]
+//=== CLI
+
+* In CLI 0.21.0, support for all `v1alpha1` resources for `clustertask`, `task`, `taskrun`, `pipeline`, and `pipelinerun` commands are deprecated. These resources are now deprecated and will be removed in a future release.
+
+//[id="deprecated-tekton-0-16-0-1-6_{context}"]
+//=== Tekton Triggers
+
+* In Tekton Triggers v0.16.0, the redundant `status` label is removed from the metrics for the `EventListener` resource.
+//  (link:https://github.com/tektoncd/triggers/pull/1166[#1166])
++
+[IMPORTANT]
+====
+Breaking change: The `status` label has been removed from the `eventlistener_http_duration_seconds_*` metric.
+Remove queries that are based on the `status` label.
+====
+
+* With the current release, the `v1alpha1` features are now deprecated and will be removed in a future release. With this update, you can begin using the `v1beta1` Go API types instead. Triggers now supports the `v1beta1` API version.
+//  (link:https://github.com/tektoncd/triggers/pull/1103[#1103])
+
+* With the current release, the `EventListener` resource sends a response before the triggers finish processing.
+//  (link:https://github.com/tektoncd/triggers/pull/1132[#1132])
++
+[IMPORTANT]
+====
+Breaking change: With this change, the `EventListener` resource stops responding with a `201 Created` status code when it creates resources. Instead, it responds with a `202 Accepted` response code.
+====
+
+* The current release removes the `podTemplate` field from the `EventListener` resource.
+//  (link:https://github.com/tektoncd/triggers/pull/1118[#1118])
++
+[IMPORTANT]
+====
+Breaking change: The `podTemplate` field, which was deprecated as part of link:https://github.com/tektoncd/triggers/pull/1100[#1100], has been removed.
+====
+
+* The current release removes the deprecated `replicas` field from the specification for the `EventListener` resource.
+//  (link:https://github.com/tektoncd/triggers/pull/1113[#1113])
++
+[IMPORTANT]
+====
+Breaking change: The deprecated `replicas` field has been removed.
+====
+
+//[id="deprecated-features-pipelines-operator-1-6_{context}"]
+//=== {pipelines-title} Operator
+
+* In {pipelines-title} 1.6, the values of `HOME="/tekton/home"` and `workingDir="/workspace"` are removed from the `Steps` specification.
++
+Instead, {pipelines-title} sets `HOME` and `workingDir` to the values defined by your `Steps` containers. You can override these values in your `Steps` specification.
```

Comment:
defined by the containers of your steps.
your `Steps` specification/the specification of your steps.